src/index.js

```diff
@@ -198,6 +201,12 @@ const InventoryRouting = (props) => {
   );
 };
 
+InventoryRouting.eventHandler = (event) => {
+  if ([coreEvents.LOGIN, 'SWITCH_ACTIVE_AFFILIATION'].includes(event)) {
+    sessionStorage.clear();
```

Comment:
It's dangerous to clear all storage on login. Its purpose is to store data between page visits and if we clear it - it defeats the purpose and potentially introduces more bugs
With this event handler approach because there is just one handler per module - we need to somehow propagate these events to other parts of the app

Reply:
> It's dangerous to clear all storage on login. Its purpose is to store data between page visits and if we clear it - it defeats the purpose and potentially introduces more bugs.

We need to avoid having the search terms after logout then login and after switching affiliation, so I'm going to clear only search terms in the next commit.

> With this event handler approach because there is just one handler per module - we need to somehow propagate these events to other parts of the app

We can process these events directly here, and if any case occurs, we can create an EventEmitter.